test/tools/metal/welder.py

```diff
@@ -32,18 +33,30 @@ def set_logger(self, level):
         return logger
 
     def create_all(self):
+        facility = None
+        try:
+            facility = self.check_capacity()
```

Comment:
Can we print some debug messages about what facility will be used? It's a useful information and in case Equinix has for example networking issues in one of their facilities we can figure out if the test failure is not related to our changes.

Reply:
yep good idea 👍